tests/utils/deployment/deployment.go

```diff
@@ -98,6 +98,9 @@ func RedefineWithMultus(deployment *appsv1.Deployment, nadNames []string) *appsv
 func RedefineWithReplicaNumber(deployment *appsv1.Deployment, replicasNumber int32) {
 	deployment.Spec.Replicas = pointer.Int32(replicasNumber)
 }
+func AppendServiceAccount(deployment *appsv1.Deployment, serviceAccountName string) {
+	deployment.Spec.Template.Spec.ServiceAccountName = "serviceAccountName"
```

Comment:
Ran these tests on my cluster and the negative case fails with:
`failed to create deployment "accesscontroldeployment" (ns accesscontrol-tests): Deployment.apps "accesscontroldeployment" is invalid: spec.template.spec.serviceAccountName: Invalid value: "serviceAccountName": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')`
so I think this value needs to be changed to something like "service-account-name"